source/mongorestore.txt

```diff
@@ -10,6 +10,9 @@
 
 .. program:: mongorestore
 
+.. meta:: 
+  :description: ``mongorestore`` is a database tool that loads data from either a binary database dump created by mongodump or the standard input into a MongoDB instance.
```

Comment:
I don't think we want text formatting in these descriptions.

```suggestion
  :description: mongorestore is a MongoDB database tool that loads data from either a binary database dump created by mongodump or the standard input into a MongoDB instance.
```

Reply:
Done! I removed the `` but am not going to add MongoDB to the intro of the sentence to avoid redundancy for SEO purposes.